airbyte-integrations/connectors/source-intercom/source_intercom/source.py

```diff
@@ -284,6 +285,7 @@ class Conversations(IncrementalIntercomStream):
     Endpoint: https://api.intercom.io/conversations
     """
 
+    use_cache = True
```

Comment:
@lgomezm I added this to avoid recalling the `conversation` endpoint when consuming  child stream `ConverationParts`